### DISCUSSION TITLE
Font Change in al-folio theme

### DISCUSSION THREAD

**Discussion by sribooshan** [alshedivat/al-folio#265] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
It is not a problem, I would like to change the font used in my al-folio themed personal site. (I think it currently uses Roboto)

**Describe the solution you'd like**
I want to know how to change the font preference in the `source` branch. For example, I would like to use SF Pro or Helvetica.

**Comment by jxnm** [comment, marked as answer]
I was able to change font family in the _base.scss file, in the _sass directory

`/*******************************************************************************
 * Styles for the base elements of the theme.
 ******************************************************************************/

// Typography

p, h1, h2, h3, h4, h5, h6, em, div, span, strong {
  color: var(--global-text-color);
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
}`

**Reply by sribooshan** [reply]
Thank you! It was useful.

**Comment by JulianStreyczek** [comment]
**As an addition:** If you don't want to use one of the (very few and ugly) standard fonts, you can very easily add cool fonts via [Google Fonts](https://fonts.google.com/). In fact, the default font in al-folio - "Roboto" - is imported via Google Fonts as well, in this line:

https://github.com/alshedivat/al-folio/blob/04f526db5d303aa3ce8c5e62089b7b6a0862f6a9/_includes/head.html#L34

If you want to change the font, you can proceed as follows:

1. Select a font (and several of its styles) from [Google Fonts](https://fonts.google.com/)
2. Open the "Selected Families" sidebar (top-right icon). It will show you two code snippets.
3. Copy the first code snippet somewhere into _includes/head.html
4. Copy the second code snippet into _sass/_base.scss right here (as in the above answer):
https://github.com/alshedivat/al-folio/blob/04f526db5d303aa3ce8c5e62089b7b6a0862f6a9/_sass/_base.scss#L5-L9

**Reply by sribooshan** [reply]
Thanks Julian, I used your method to change fonts for my website previously.

**Comment by arashgmn** [comment]
Thanks, @JulianStreyczek . This solution works but also changes the code block's font. Is there a way to prevent that?

**Reply by george-gca** [reply]
Simply add the desired font to your [_sass/_base.scss](https://github.com/alshedivat/al-folio/blob/c824d128673c000ef9f0e0e77a0ae30d2c4b4568/_sass/_base.scss#L744) code class.
